docs/get-started/migrating-from-sequel-pro.md

```diff
@@ -7,43 +7,36 @@
 
 <hr>
 
-### Migrating to Sequel Ace from Sequel Pro
+### Migrating to `Sequel Ace` from `Sequel Pro`
 
 #### Migrating Connection Favorites
 _The following is based on [this medium article](https://medium.com/@harrybailey/migration-from-sequel-pro-to-sequel-ace-c6a579399c90):_
-1. Install Sequel Ace
 
-2. Quit both Sequel Pro and Sequel Ace
+:exclamation: This process will replace (you will lose) any connections already in `Sequel Ace`, so you may want to make a note of those details first.
 
-3. NB: This process will replace (you will lose) any connections already in Sequel Ace, so you may want to make a note of those details first
+1. [Install `Sequel Ace`](https://sequel-ace.com)
 
-4. Copy
-~/Library/Preferences/com.sequelpro.SequelPro.plist
-to
-~/Library/Containers/com.sequel-ace.sequel-ace/Data/Library/Preferences/com.sequel-ace.sequel-ace.plist
+2. Quit both `Sequel Pro` and `Sequel Ace`
 
-5. Copy
-~/Library/Application\ Support/Sequel\ Pro/Data/Favorites.plist
-to
-~/Library/Containers/com.sequel-ace.sequel-ace/Data/Library/Application\ Support/Sequel\ Ace/Data/Favorites.plist
+3. Copy Settings via `cp ~/Library/Preferences/com.sequelpro.SequelPro.plist ~/Library/Containers/com.sequel-ace.sequel-ace/Data/Library/Preferences/com.sequel-ace.sequel-ace.plist1`
```

Comment:
Looks like there was a 1 accidentally added to the end of this line?

Reply:
true ; good catch